streams/ru.m3u

```diff
@@ -121,10 +124,12 @@ http://strm.yandex.ru/kal/europaplus/europaplus0.m3u8
 http://infochhdcdn.trkeurasia.ru/orsk-infochhd/infochhd/playlist.m3u8
 #EXTINF:-1 tvg-id="Evraziya.ru" status="online",Евразия (Орск) (720p)
 https://infochh.trkeurasia.ru/hlsinfoch/infochhd.m3u8
-#EXTINF:-1 tvg-id="Evronovosti.ru" status="online",Евроновости (540p)
+#EXTINF:-1 tvg-id="Rossiya24.ru" status="online",Россия 24 (540p)
 http://evronovosti.mediacdn.ru/sr1/evronovosti/playlist.m3u8
-#EXTINF:-1 tvg-id="Evronovosti.ru" status="error",Евроновости (720p) [Geo-blocked]
+#EXTINF:-1 tvg-id="Rossiya24.ru" status="error",Россия 24 (720p) [Geo-blocked]
 https://strm.yandex.ru/kal/euronews_supres/euronews_supres0.m3u8
+#EXTINF:-1 tvg-id="Evronovosti.ru" status="online",Евроновости (576p)
+http://185.32.44.154/euronews/tracks-v1a1/mono.m3u8
```

Comment:
```suggestion
http://185.32.44.154/euronews/index.m3u8
```
+ Evronovosti = Euronews Russia, Euronews is French in it's base, should be located in fr.m3u.

Reply:
well, it was in the ru.m3u list, I just fixed the url to work again